res/menu/upload_files.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+      xmlns:app="http://schemas.android.com/apk/res-auto">
```

Comment:
@Andy Scherzinger ok



Allan@dingyantec.com
 
From: Andy Scherzinger
Date: 2015-12-18 17:01
To: owncloud/android
CC: Allan
Subject: Re: [android] Add select all file when upload file to remote (#1342)
In res/menu/upload_files.xml:
> @@ -0,0 +1,11 @@
> +<?xml version="1.0" encoding="utf-8"?>
> +<menu xmlns:android="http://schemas.android.com/apk/res/android"
> +      xmlns:app="http://schemas.android.com/apk/res-auto">

@xxNull-lsk Can you then bring back app:... ? 
—
Reply to this email directly or view it on GitHub.